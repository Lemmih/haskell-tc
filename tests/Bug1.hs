module Bug1 where

unit = (\x -> x) ()
