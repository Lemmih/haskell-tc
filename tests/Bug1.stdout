Bindings:
:: ()
  unit = (\[4mx[0m -> x) ()

Proofs:
coercion: ()
  [4munit = (\x -> x) ()[0m
coercion: ()
  unit = (\x -> [4mx[0m) ()
